nerfstudio/configs/method_configs.py

```diff
@@ -137,6 +138,37 @@
     vis="viewer",
 )
 
+method_configs["volinga"] = TrainerConfig(
+    method_name="volinga",
+    steps_per_eval_batch=500,
+    steps_per_save=2000,
+    max_num_iterations=30000,
+    mixed_precision=True,
+    pipeline=VanillaPipelineConfig(
+        datamanager=VanillaDataManagerConfig(
+            dataparser=NerfstudioDataParserConfig(),
+            train_num_rays_per_batch=4096,
+            eval_num_rays_per_batch=4096,
+            camera_optimizer=CameraOptimizerConfig(
+                mode="SO3xR3", optimizer=AdamOptimizerConfig(lr=6e-4, eps=1e-8, weight_decay=1e-2)
+            ),
+        ),
+        model=VolingaModelConfig(eval_num_rays_per_chunk=1 << 15),
```

Comment:
Looks as though the Volinga model config uses Nerfacto model with specific parameters?  If so, may be cleaner to use existing NerfactoModelConfig with desired Volinga settings, similar to how instant-ngp-bounded leverages instant-ngp.